src/SAWScript/Interpreter.hs

```diff
@@ -345,11 +347,28 @@ interpretStmt printBinds stmt =
     SS.StmtTypedef _ name ty   -> do rw <- getTopLevelRW
                                      putTopLevelRW $ addTypedef (getVal name) ty rw
 
+writeVerificationSummary :: TopLevel ()
+writeVerificationSummary = do
+  do values <- rwProofs <$> getTopLevelRW
+     let jspecs  = [ s | VJVMMethodSpec s <- values ]
+         lspecs  = [ s | VLLVMCrucibleMethodSpec s <- values ]
+         thms    = [ t | VTheorem t <- values ]
+         summary = computeVerificationSummary jspecs lspecs thms
+     opts <- roOptions <$> getTopLevelRO
+     dir <- roInitWorkDir <$> getTopLevelRO
+     case summaryFile opts of
+       Nothing -> return ()
+       Just f ->
+         let f' = if hasDrive f then f else dir </> f
+          in io $ writeFile f' $ prettyVerificationSummary summary
```

Comment:
I expect `writeFile` to work with both absolute and relative paths. Is that not the case?

Reply:
Yes, but the current directory is set to the one in which the `.saw` file being process resides (in `processFile` in `Interpreter.hs`). I think that the user would instead expect the file to be written in the directory from which the command was invoked.